src/components/handbook-navigation/TocPagination.js

```diff
@@ -22,7 +22,7 @@ const TocPagination = () => {
       ) : null}
 
       {currentPage < content.length - 1 ? (
-        <Button $primary $url={content[currentPage + 1]?.link}>
+        <Button $primary $url={content[currentPage + 1]?.link} onClick={() => console.log("button is clicked")}>
```

Comment:
Remove the `onCLick` statment from the button
```suggestion
        <Button $primary $url={content[currentPage + 1]?.link}>
```